### PR TITLE
docs(vscode): Add onboarding overview to readme file

### PR DESCRIPTION
**Pull request by wsilveiranz**
- **Please check if the PR fulfills these requirements**

* [X] The commit message follows our guidelines
* [ ] Tests for the changes have been added (for bug fixes/features)
* [X] Docs have been added / updated (for bug fixes / features)

- **What kind of change does this PR introduce?** Doc update - VS Code Extension Code readme.md file

- **What is the current behavior?** N/A - doc update 

- **What is the new behavior (if this is a feature change)?** - N/A - doc update

- **Does this PR introduce a breaking change?** No

- **Please Include Screenshots or Videos of the intended change**: N/A

AB#26000724

### CODE REVIEW

File: apps/vs-code-designer/src/README.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ In Visual Studio Code, you can use the Azure Logic Apps (Standard) extension to
 
 > Sign up today for your free Azure account and receive 12 months of free popular services, $200 free credit and 25+ always free services 👉 [Start Free](https://azure.microsoft.com/free/open-source).
 
+## Azure Logic Apps - Improved Onboarding Experience
+
+Starting with version 2.81.5, the Azure Logic Apps (Standard) extension for Visual Studio Code includes a dependency installer that automatically installs all the required dependencies in a new binary folder and leaves any existing dependencies unchanged. For more information, see [Get started more easily with the Azure Logic Apps (Standard) extension for Visual Studio Code](https://techcommunity.microsoft.com/t5/azure-integration-services-blog/making-it-easy-to-get-started-with-the-azure-logic-apps-standard/ba-p/3979643).
```
Comment (ccastrotrejo) [inline apps/vs-code-designer/src/README.md:9]: @ecfan Could you please review this text? @wsilveiranz We will problably need a go link for the documentation?

**Review by ccastrotrejo — APPROVED** [review]

**Comment by wsilveiranz** [comment]
@ccastrotrejo - link updated.

File: apps/vs-code-designer/src/README.md (hunk 1 above)
Comment (ecfan) [reply apps/vs-code-designer/src/README.md]: Looks good to me as this paragraph came from the official doc: https://learn.microsoft.com/azure/logic-apps/create-single-tenant-workflows-visual-studio-code

Comment (ecfan) [reply apps/vs-code-designer/src/README.md]: #assign:ecfan

**Pull request merged**